resolver_winxp.c

```diff
@@ -263,15 +226,17 @@ bool proxy_resolver_winxp_global_cleanup(void) {
 }
 
 proxy_resolver_i_s *proxy_resolver_winxp_get_interface(void) {
-    static proxy_resolver_i_s proxy_resolver_winxp_i = {proxy_resolver_winxp_get_proxies_for_url,
-                                                        proxy_resolver_winxp_get_list,
-                                                        proxy_resolver_winxp_get_error,
-                                                        proxy_resolver_winxp_wait,
-                                                        proxy_resolver_winxp_cancel,
-                                                        proxy_resolver_winxp_create,
-                                                        proxy_resolver_winxp_delete,
-                                                        proxy_resolver_winxp_is_async,
-                                                        proxy_resolver_winxp_global_init,
-                                                        proxy_resolver_winxp_global_cleanup};
+    static proxy_resolver_i_s proxy_resolver_winxp_i = {
+        proxy_resolver_winxp_get_proxies_for_url,
+        proxy_resolver_winxp_get_list,
+        proxy_resolver_winxp_get_error,
+        proxy_resolver_winxp_wait,
+        proxy_resolver_winxp_cancel,
+        proxy_resolver_winxp_create,
+        proxy_resolver_winxp_delete,
+        false /* get_proxies_for_url should be spooled to another thread */,
+        false /* get_proxies_for_url does not take into account system config */,
```

Comment:
The comas after the comments do not look nice.
```suggestion
        false,  // get_proxies_for_url should be spooled to another thread
        false,  // get_proxies_for_url does not take into account system config
```

Reply:
Fixed.